glyph/src/font.rs

```diff
@@ -100,6 +100,9 @@ pub trait Font {
     /// Scaling can be done with [`as_scaled`](Self::as_scaled).
     fn line_gap_unscaled(&self) -> f32;
 
+    /// The slant angle of the font.
+    fn italic_angle(&self) -> f32;
```

Comment:
An additional trait method makes this a breaking change for external `Font` implementations. We've prevented similar breaks by adding a default impl to such new methods.

```suggestion
    fn italic_angle(&self) -> f32 {
        // Note: default impl prevents this method from breaking external
        //       Font impls written before introduction
        0.0
    }
```